src/main/java/org/folio/consortia/service/impl/PublicationServiceImpl.java

```diff
@@ -265,4 +267,28 @@ private PublicationResponse buildPublicationResponse(UUID publicationId) {
       .status(PublicationStatus.IN_PROGRESS);
   }
 
+
+  @Override
+  public PublicationResultCollection getPublicationResults(UUID consortiumId, UUID publicationId){
+    log.debug("getPublicationResults:: Trying to retrieve publication results by consortiumId: {} and publicationId id: {}", consortiumId, publicationId);
+
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+    var publicationStatusEntity = publicationStatusRepository.findById(publicationId)
+      .orElseThrow(() -> new ResourceNotFoundException("publicationId", String.valueOf(publicationId)));
+
+    var ptrEntities = publicationTenantRequestRepository.findByPcStateId(publicationId, PageRequest.of(0, Integer.MAX_VALUE));
+    log.info("getPublicationResults:: Found {} of {} expected tenant request records", ptrEntities.getTotalElements(), publicationStatusEntity.getTotalRecords());
```

Comment:
maybe also log pblication and consortium ids to make this info entry more useful?